Tests/UnitTests/RequestBuilderTests.swift

```diff
@@ -31,7 +31,21 @@ class RequestBuilderTests: XCTestCase {
     func testGetRequestPayload_allParameters_verifyMetadata() {
         let request = RequestBuilder()
         request.enableResponseStreaming(recordSeparator: "A", lineFeed: "B")
-        request.experienceCloudId = "ecid"
+        guard let identityMapData = """
```

Comment:
Same comment in this file about adding "identityMap" to the test shared states.

Reply:
Same as my above comment, we are populating the "identityMap" field on the request.

Reply:
Updated this. I got dynamic encoding to work.